projects/fsastorefrontlib/src/cms-components/checkout/checkout.module.ts

```diff
@@ -216,8 +217,9 @@ const routes: Routes = [
         QuoteReviewFlex: {
           component: QuoteReviewComponent,
         },
-        PaymentDetailsFlex: {
+        CheckoutPaymentDetails: {
```

Comment:
@bojanjevtic-spa I see now that you replaced `PaymentDetailsFlex ` with `CheckoutPaymentDetails `. Should this stay as it is? With `CheckoutPaymentDetails ` flex component works fine in both cases (when `digitalPayments` is true and false).

Reply:
One question. If you say now that digitalPayments is false, is our mocked payment step going to be displayed (invoice/credit card option)? 
Payment-method body slot contains PaymentDetailsFlex, not CheckoutPaymentDetails component, right?